yksgi/src/lib.rs

```diff
@@ -199,12 +200,36 @@ impl SGInterp {
                     self.switch();
                     continue;
                 }
+                LLVMOpcode::LLVMSub => self.sub(),
+                LLVMOpcode::LLVMSelect => self.select(),
                 _ => todo!("{:?}", self.pc.as_str()),
             }
             self.pc = Value::new(LLVMGetNextInstruction(self.pc.get()));
         }
     }
 
+    fn select(&mut self) {
+        let cond = self.pc.get_operand(0);
+        let op_true = self.pc.get_operand(1);
+        let op_false = self.pc.get_operand(2);
+
+        if op_true.get_type().is_vector() || op_false.get_type().is_vector() {
+            todo!(
```

Comment:
Is @ltratt okay with strings in todo's these days? I've been trying not to use them (even though one in `interpet()` would be useful) to avoid his wrath.

Reply:
You're making me out to be a monster -- just because I'm big, green, and hairy doesn't mean I'm a monster ;)

I've always been fine with strings in `todo`s because they still cause the program to terminate (they just give you more information about *why* the program terminated).

Reply:
@ptersilie Binary size? You have an active imagination :p